lib/pusher-js/core/options.dart

```diff
@@ -80,7 +80,7 @@ abstract class Options {
       num httpPort,
       num httpsPort,
       bool ignoreNullOrigin,
-  //    nacl nacl,
+      //    nacl nacl,
```

Comment:
What do you think about removing this comment?

Reply:
done